include/z64save.h

```diff
@@ -717,8 +719,12 @@ typedef enum {
 #define WEEKEVENTREG_20_80 PACK_WEEKEVENTREG_FLAG(20, 0x80)
 #define WEEKEVENTREG_21_01 PACK_WEEKEVENTREG_FLAG(21, 0x01)
 #define WEEKEVENTREG_21_02 PACK_WEEKEVENTREG_FLAG(21, 0x02)
-#define WEEKEVENTREG_21_04 PACK_WEEKEVENTREG_FLAG(21, 0x04)
-#define WEEKEVENTREG_21_08 PACK_WEEKEVENTREG_FLAG(21, 0x08)
+
+// Player has spoken to Goron Graveyard's gravemaker while in Goron form.
+#define WEEKEVENTREG_TALKED_TO_GORON_GRAVEMAKER_AS_GORON PACK_WEEKEVENTREG_FLAG(21, 0x04)
```

Comment:
```suggestion
#define WEEKEVENTREG_TALKED_GORON_GRAVEMAKER_AS_GORON PACK_WEEKEVENTREG_FLAG(21, 0x04)
```
The style for the many "talked-to-actor" flags that are set is `WEEKEVENTREG_TALKED_ACTOR_CIRCUMSTANCE`, so you're style was close, just drop the `TO` (these names can get long)